### PR TITLE
New features added

### PR DESCRIPTION
**Pull request by yanis60**
Hi,
I've added some new features, such as

1- Border for page.
2- GroupHeader has a sortfieldname property.
3- Guide lines for alignement.
4- Improved context menu for aligning items

### CODE REVIEW

File: 3rdparty/object_script.QtZint.Release
Comment (Jihadist) [inline 3rdparty/object_script.QtZint.Release:1]: Please do not include your local files

**Comment by yanis60** [comment]
i pulled the latest commit from original repo before pushing it

File: 3rdparty/object_script.QtZint.Release
Comment (fralx) [reply 3rdparty/object_script.QtZint.Release:1]: @yanis60 Hi,  Your commit contains auto generated files such as:
[3rdparty/object_script.QtZint.Release](https://github.com/fralx/LimeReport/pull/461/files#diff-c679fc16ab3101ac5e25ab06c530a2bcd24a1e2fed1b3751dc312f8341953667)
[3rdparty/win32/release/moc/moc_predefs.h](https://github.com/fralx/LimeReport/pull/461/files#diff-56730d8afa6a51742f13ac5ad8bdb5bd765d14c5e13a7973c03dc6d47056ae3c)
and so on. Remove it please

File: CMakeLists.txt
Comment (Jihadist) [inline CMakeLists.txt:1]: Please do not delete cmakelists :)

File: OKQmAj.json
Comment (Jihadist) [inline OKQmAj.json:1]: Should be deleted

File: limereport/win32/debug/moc/mocinclude.opt
Comment (Jihadist) [inline limereport/win32/debug/moc/mocinclude.opt:1]: There are your local files here

File: CMakeLists.txt
Comment (yanis60) [reply CMakeLists.txt:1]: sorry i'm not good in github stuffs, i know just how to commit and push and pull :)

File: .vscode/settings.json
Comment (fralx) [inline .vscode/settings.json:1]: Hi. This file have to be removed

File: Nouveau dossier/lrbasedesignintf.cpp
Comment (fralx) [inline Nouveau dossier/lrbasedesignintf.cpp:1]: I guess this folder  [Nouveau dossier](https://github.com/fralx/LimeReport/pull/461/files#diff-0fd63145849afefd4de7e771295a8d653cf4e7a3091b10ba76fc5a4c7c7669f5) should be removed as redundant

File: .vscode/settings.json
Comment (yanis60) [reply .vscode/settings.json:1]: i apology if some files are commited

File: limereport/win32/release/moc/mocinclude.opt
Comment (fralx) [inline limereport/win32/release/moc/mocinclude.opt:1]: The files in this folder limereport/win32 are autogerated

File: lrdview/release/lrdview.exe
Comment (fralx) [inline lrdview/release/lrdview.exe:1]: This file have to removed  from the  commit

File: lrdview/win32/debug/moc/mocinclude.opt
Comment (fralx) [inline lrdview/win32/debug/moc/mocinclude.opt:1]: This file have to removed from the commit

File: lrdview/win32/release/moc/mocinclude.opt
Comment (fralx) [inline lrdview/win32/release/moc/mocinclude.opt:1]: This file have to removed from  the commit

File: translations/limereport_ar.ts
Comment (fralx) [inline translations/limereport_ar.ts:1]: if you didn't change the translation, you shouldn't include it in the commit

File: translations/limereport_es.ts
Comment (fralx) [inline translations/limereport_es.ts:1]: if you didn't change the translation, you shouldn't include it in the commit

File: translations/limereport_zh.ts
Comment (fralx) [inline translations/limereport_zh.ts:1]: if you didn't change the translation, you shouldn't include it in the commit

File: translations/limereport_ru.ts
Comment (fralx) [inline translations/limereport_ru.ts:1]: if you didn't change the translation, you shouldn't include it in the commit

File: translations/limereport_ru.qm
Comment (fralx) [inline translations/limereport_ru.qm:1]: if you didn't change the translation, you shouldn't include it in the commit

File: translations/limereport_pl.ts
Comment (fralx) [inline translations/limereport_pl.ts:1]: if you didn't change the translation, you shouldn't include it in the commit

File: translations/limereport_fr.ts
Comment (fralx) [inline translations/limereport_fr.ts:1]: if you didn't change the translation, you shouldn't include it in the commit

File: include/lrrenderengine.h
Comment (fralx) [inline include/lrrenderengine.h:1]: semicolon is redundant

File: .vscode/settings.json
Comment (fralx) [reply .vscode/settings.json:1]: This file still exists in the commit

File: README.md
Comment (fralx) [inline README.md:1]: This changes should be rollback

**Review by fralx — COMMENTED** [review]
Hi! Some files need to be removed

File: translations/limereport_ar.ts
Comment (fralx) [inline translations/limereport_ar.ts:1]: You should't delete this file you have to just not include it in commit. The thanslations files have to return back.

Comment (yanis60) [reply translations/limereport_ar.ts:1]: i'm really sorry for those mistakes this is becasuse of .gitignore i guess

File: CMakeLists.txt
Comment (Jihadist) [reply CMakeLists.txt:1]: Please do not commit changes that does not really change anything.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 
 # LimeReport v1.7.5 [![Build status](https://ci.appveyor.com/api/projects/status/wna5429pix7ilcmo/branch/master?svg=true)](https://ci.appveyor.com/project/fralx/limereport/branch/master) [![Codacy Badge](https://api.codacy.com/project/badge/Grade/bc31412ea4814f30825b5ed3723e9a70)](https://app.codacy.com/app/fralx/LimeReport?utm_source=github.com&utm_medium=referral&utm_content=fralx/LimeReport&utm_campaign=Badge_Grade_Dashboard) ![Cmake Build Status](https://github.com/fralx/limereport/actions/workflows/cmake.yml/badge.svg)
```
Comment (Jihadist) [inline README.md:3]: Also here

File: common.pri [hunk 2]
```diff
@@ -32,7 +32,7 @@ CONFIG(easy_profiler) {
     message(EasyProfiler)
     INCLUDEPATH *= $$PWD/3rdparty/easyprofiler/easy_profiler_core/include
     DEPENDPATH *= $$PWD/3rdparty/easyprofiler/easy_profiler_core/include
-    unix|win32: LIBS *= -L$$PWD/3rdparty/easyprofiler/build/bin/ -leasy_profiler
```
Comment (Jihadist) [inline common.pri:35]: Also here

**Review by Jihadist — CHANGES_REQUESTED** [review]
Although it looks much better, but it had not completed yet

File: common.pri (hunk 2 above)
Comment (yanis60) [reply common.pri:35]: just spaces that has been removed here

**Review by fralx — COMMENTED** [review]
Please explain why sort field name property is needed?

**Pull request closed without merging**